packages/web-pkg/src/composables/actions/spaces/useSpaceActionsSetIcon.ts

```diff
@@ -31,6 +31,7 @@ export const useSpaceActionsSetIcon = () => {
       title: $gettext('Set icon for %{space}', { space: resources[0].name }),
       hideConfirmButton: true,
       customComponent: EmojiPickerModal,
+      focusTrapActive: false,
```

Comment:
Ooooh wait. The focus trap is required for all modals, otherwise it's an a11y violation. Why is this line required?

Reply:
This is JUST for the emoji picker modal, the emoji picker is not compatible with the shadowRoot implementation of the emoji picker and so the emoji picker handles the focus by itself. What does the focus trap in addition ?

Reply:
Ok understood what the trap should do, somehow the focus trap breaks the emoji picker ...

Reply:
Got it, I don't have a solution to this on hand rn

Reply:
Fixed